### DISCUSSION TITLE
In Janus Graph db how to get notified about new vertex/edges?

### DISCUSSION THREAD

**Discussion by mkj-git** [JanusGraph/janusgraph#4740] [General]
In Janus graph db, is there a way to get notified when a new vertex/edge is added/removed/updated?

One solution is this https://docs.janusgraph.org/advanced-topics/transaction-log/#log-configuration But one limitation I can see is that logIdentifier is required, can we do it without that?

**Comment by hadoopmarc** [comment]
Indeed, a logIdentifier is required, see: https://github.com/JanusGraph/janusgraph/blob/80ef1d91f7ddc0da5b9f965016b51fee2392b8b5/janusgraph-core/src/main/java/org/janusgraph/graphdb/log/StandardLogProcessorFramework.java#L128

Another thing you could do is add creation timestamps to your graph elements and index them. That makes it possible to periodically query for the latest elements.
